kubeflow/trainer/experimental/backends/kubernetes.py

```
@@ -0,0 +1,265 @@
+            Union[
+                types.CustomTrainer,
+                types.CustomTrainerContainer,
+                types.BuiltinTrainer,
+                TransformersTrainer,
+            ]
+        ] = None,
+        options: Optional[list] = None,
+    ) -> str:
+        """Train with experimental trainer support."""
+        if trainer is None:
+            raise ValueError("Trainer cannot be None")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)